### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by mindtrip101** [dashpay/p2pool-dash#42]
Is anyone else having a memory leak? Seems as if the pools will stay up for about 4-6 hours before they reach a to high memory usage error.

**Comment by djames007** [comment]
Yes, this is also happening for me with version 15.0-105-g7d31661 (jtoomim/segwit).   I have 16gb of ram and upping the memory usage in bitcoin-qt is only workaround. 8gb of memory devotion in biotcoin 0.15 will get me about 8 days before the memory core dumps.
